src/7-to-8/caveats.rst

```diff
@@ -5,14 +5,6 @@ There are a few Cylc 7 features which do not yet have direct replacements in
 Cylc 8. These features will be re-implemented in future releases.
 
 
-Cylc Flow
----------
-
-Multiple Flows
-   The new :term:`scheduler` can manage multiple flows in the workflow graph.
-   Commands and options for interacting with flows are still being refined.
-
-
```

Comment:
Off topic, but I think this can be removed now that flows are supported by:

* stop
* trigger
* set
* remove